src/server/pfs/server/driver_v2.go

```diff
@@ -536,3 +581,81 @@ func (d *driverV2) globFileV2(pachClient *client.APIClient, commit *pfs.Commit,
 		return f(fr.Info())
 	})
 }
+
+func (d *driverV2) copyFile(pachClient *client.APIClient, src *pfs.File, dst *pfs.File, overwrite bool) (retErr error) {
+	ctx := pachClient.Ctx()
+	if overwrite {
+		// TODO: after delete merging is sorted out add overwrite support
+		return errors.New("overwrite not yet supported")
+	}
+	srcPrefix := cleanPath(src.Path)
+	dstPrefix := cleanPath(dst.Path)
+	pathTransform := func(x string) string {
+		if !strings.HasPrefix(x, srcPrefix) {
```

Comment:
I think this would probably be cleaner if it used a relative path function (such as `filepath.Rel`) rather than checking a prefix then appending everything after it. Since we don't support Windows, I think this might be the better approach from a readability standpoint.